tests/benchdnn/inputs/matmul/test_matmul_all

```diff
@@ -74,3 +74,22 @@
 
 # matmul with strides
 --batch=harness_matmul_strides
+
+# 4d
+
+--reset
+--cfg=f32
+--stag=abcd,abdc --wtag=abcd,abdc --dtag=abcd,abdc
+--bia_dt=undef,f32
+
+--attr-oscale=common:2.25
+--attr-post-ops=,relu
+--batch=shapes_4d
+
+--attr-oscale=common:2.25
+--attr-post-ops=,sum
+--batch=shapes_4d
+
+--attr-oscale=common:2.25
+--attr-post-ops=sum+relu+add:u8,add:f32:per_tensor+sub:f32:per_oc,add:f32:per_dim_2
+--batch=shapes_4d
```

Comment:
```suggestion
--attr-oscale=common:2.25
--attr-post-ops=,relu,sum,sum+relu+add:u8
--batch=shapes_4d
```
I don't see a big need in per_tensor post-ops or long chains. It will likely fall back in ref code anyway.